packages/-ember-decorators/tests/dummy/config/environment.js

```diff
@@ -44,6 +45,8 @@ module.exports = function(environment) {
   }
 
   if (environment === 'production') {
+    // Allow ember-cli-addon-docs to update the rootURL in compiled assets
+    ENV.rootURL = 'ADDON_DOCS_ROOT_URL';
     ENV.locationType = 'hash';
     ENV.rootURL = '/ember-decorators/';
```

Comment:
Looks like these `rootURL`s are going to conflict

Reply:
good catch, ty!